cosmo_tester/test_suites/test_blueprints/test_existing_vm.py

```diff
@@ -99,19 +104,28 @@ def create_keypair_and_copy_to_manager(self,
         fabric.api.put(local_path=key_file,
                        remote_path=remote_key_path)
 
+    def get_nics_by_network_name(self, network_name, neutron_client):
+        nets = neutron_client.list_networks()
```

Comment:
FYI, you could pass keyword arguments into this call and have the server filter out the networks it returns, so instead of looking for the one with the correct name on the client side, you could get back only the networks with the given name.